front/app/hooks/useExceedsSeats.ts

```diff
@@ -1,6 +1,6 @@
 import useSeats from 'api/seats/useSeats';
 import { isNil } from 'utils/helperUtils';
-import { useAvailableSeats } from './useAvailableSeats';
+import useTotalSeats from './useTotalSeats';
 
 interface ReturnFunctionParameters {
   newlyAddedAdminsNumber: number;
```

Comment:
Any chance we could make these optional? From some of the use cases, there are scenarios where we are only interested in one of them